<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimersDisplayWidget</class>
 <widget class="QWidget" name="TimersDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>113</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="time-admin"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QLineEdit" name="editSearch"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="buttonClear">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Clear</string>
     </property>
     <property name="icon">
      <iconset theme="edit-clear"/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QListView" name="timersView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelIconSearch"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClear</sender>
   <signal>clicked()</signal>
   <receiver>editSearch</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
